Description: Mark tests XFAIL that fail due to Pandas datetime
 See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=973821
 Pandas in Debian > 1.0.3, and 1.1.4 (current) causes failures in datetime.
 Mark them XFAIL here for the moment
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=973115
Forwarded: yes
Last-Updated: 2021-09-02

--- a/tests/test_40_xarray_to_grib_regular_ll.py
+++ b/tests/test_40_xarray_to_grib_regular_ll.py
@@ -28,7 +28,7 @@
     )
     return da
 
-
+@pytest.mark.xfail(reason="Fails on pandas > 1.0.3 in Debian")
 def test_canonical_dataarray_to_grib_with_grib_keys(
     canonic_da: xr.DataArray, tmpdir: py.path.local
 ) -> None:
@@ -37,7 +37,7 @@
     with open(str(out_path), "wb") as file:
         xarray_to_grib.canonical_dataarray_to_grib(canonic_da, file, grib_keys=grib_keys)
 
-
+@pytest.mark.xfail(reason="Fails on pandas > 1.0.3 in Debian")
 def test_canonical_dataarray_to_grib_detect_grib_keys(
     canonic_da: xr.DataArray, tmpdir: py.path.local
 ) -> None:
@@ -55,7 +55,7 @@
         with pytest.raises(ValueError):
             xarray_to_grib.canonical_dataarray_to_grib(canonic_da, file, grib_keys=grib_keys)
 
-
+@pytest.mark.xfail(reason="Fails on pandas > 1.0.3 in Debian")
 def test_canonical_dataset_to_grib(canonic_da: xr.DataArray, tmpdir: py.path.local) -> None:
     out_path = tmpdir.join("res.grib")
     canonic_ds = canonic_da.to_dataset(name="t")
@@ -65,6 +65,7 @@
     xarray_to_grib.canonical_dataset_to_grib(canonic_ds, str(out_path), no_warn=True)
 
 
+@pytest.mark.xfail(reason="Fails on pandas > 1.0.3 in Debian")
 def test_to_grib(canonic_da: xr.DataArray, tmpdir: py.path.local) -> None:
     out_path = tmpdir.join("res.grib")
     canonic_ds = canonic_da.to_dataset(name="t")
--- a/tests/test_50_sample_data.py
+++ b/tests/test_50_sample_data.py
@@ -81,6 +81,8 @@
         "t_analysis_and_fc_0",
     ],
 )
+
+@pytest.mark.xfail(reason="Fails on pandas > 1.0.3 in Debian")
 def test_canonical_dataset_to_grib(grib_name: str, tmpdir: py.path.local) -> None:
     grib_path = os.path.join(SAMPLE_DATA_FOLDER, grib_name + ".grib")
     out_path = str(tmpdir.join(grib_name + ".grib"))
